cmd/promtool/main.go

```diff
@@ -112,6 +114,18 @@ func main() {
 		"The config files to check.",
 	).Required().ExistingFiles()
 
+	checkServerHealthCmd := checkCmd.Command("health", "Check the health of a Prometheus server")
+	serverHealthURLArg := checkServerHealthCmd.Arg(
+		"url",
+		"The URL of the Prometheus server to check (e.g. http://localhost:9090)",
+	).String()
+
+	checkServerReadyCmd := checkCmd.Command("ready", "Check the readiness of a Prometheus server")
+	serverReadyURLArg := checkServerReadyCmd.Arg(
+		"url",
+		"The URL of the Prometheus server to check (e.g. http://localhost:9090)",
+	).String()
```

Comment:
In other commands (`query`, `debug`), the `url` argument is referred to as `server`. For consistency for both users and maintainers, I think we should adopt `server` here as well

Reply:
Addressed